yb-voyager/src/tgtdb/target_db_interface.go

```diff
@@ -43,6 +43,10 @@ type TargetDB interface {
 	RestoreSequences(sequencesLastValue map[string]int64) error
 	GetImportedEventsStatsForTable(tableName string, migrationUuid uuid.UUID) (*EventCounter, error)
 	GetImportedSnapshotRowCountForTable(tableName string) (int64, error)
+	GetGeneratedAlwaysIdentityColumnNamesForTable(table string) ([]string, error)
+	GetGeneratedAlwaysAsIdentityColumnsForTables(tables []string) (map[string][]string, error)
```

Comment:
Implementation of this method will be the same for all `TargetDB` interface -- iterate through all the `tables` and call the "single table name" version of the same method.

I would suggest it to define this as a separate function in the `importData.go`.